docker/docker-compose.yml

```diff
@@ -82,3 +67,72 @@ services:
       - 'traefik.http.middlewares.strip_auth_prefix.stripprefix.prefixes=/${API_VERSION}/auth'
       - 'traefik.http.middlewares.strip_auth_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.auth.middlewares=strip_auth_prefix'
+
+  ##############################################################################
+  # Third-Party Dependencies and Support Services
+  ##############################################################################
+
+  # Our main ingress router, handling all traffic on 80/443
+  nginx:
+    image: nginx:latest
+    container_name: 'nginx'
+    volumes:
+      - ../config/nginx.development.conf:/etc/nginx/nginx.conf
+    ports:
+      - 80:80
+    depends_on:
+      - traefik
+
+  # API Gateway
+  traefik:
+    image: traefik:v2.4.5
+    container_name: 'traefik'
+    command:
+      # If you need Traefik debug logs, uncomment this
+      #- '--log.level=DEBUG'
+      - '--api.insecure=false'
+      - '--providers.docker=true'
+      - '--providers.docker.exposedbydefault=false'
+      - '--entryPoints.web.address=:8888'
+    ports:
+      - '8888'
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    labels:
+      - 'traefik.enable=true'
+
+  # Redis
+  redis:
+    image: redis:latest
+    container_name: 'redis'
+    ports:
+      # In development, we allow access to redis via local port binding
+      - '6739:6739'
```

Comment:
```suggestion
      - '6379:6379'
```

Reply:
Also, we should be careful with this, it might be exposing `6379` in production since (I think), in some cases `docker-compose` doesn't override but expands (meaning it'd use this and the port settings in `production.yml`.